Zscaler/zscaler-zia/_meta/smart-descriptions.json

```
@@ -0,0 +1,72 @@
+[
+  {
+    "value": "Threat {zscaler.zia.threat.name} with category {zscaler.zia.threat.category}, class {zscaler.zia.threat.class} and event id {zscaler.zia.event_id} detected",
+    "conditions": [
+      {
+        "field": "zscaler.zia.threat.name"
+      },
+      {
+        "field": "zscaler.zia.threat.category"
+      },
+      {
+        "field": "zscaler.zia.threat.class"
+      },
+      {
+        "field": "zscaler.zia.threat.class"
+      }
+    ]
+  },
+  {
+    "value": "Threat {zscaler.zia.threat.name} detected",
+    "conditions": [
+      {
+        "field": "zscaler.zia.threat.name"
+      }
+    ]
+  },
+  {
+    "value": "{http.request.method} request with referer {http.request.referrer} have result {http.response.status_code}",
+    "conditions": [
+      {
+        "field": "http.request.method"
+      },
+      {
+        "field": "http.request.referrer"
+      },
+      {
+        "field": "http.response.status_code"
+      }
+    ]
+  },
+  {
+    "value": "Event {event.reason} on vendor {zscaler.zia.vendor}, product {zscaler.zia.product}, category {zscaler.zia.category} and result {zscaler.zia.action.result}",
+    "conditions": [
+      {
+        "field": "event.reason"
+      },
+      {
+        "field": "zscaler.zia.vendor"
+      },
+      {
+        "field": "zscaler.zia.product"
+      },
+      {
+        "field": "zscaler.zia.category"
+      },
+      {
+        "field": "zscaler.zia.action.result"
+      }
+    ]
+  },
```

For connection events

```suggestion
  {
    "value": "Connection from {user.email} to {destination.address}",
    "conditions": [
      {
        "field": "event.category",
	"value": "network"
      },
      {
        "field": "event.type",
	"value": "connection"
      },
      {
        "field": "user.email"
      },
      {
        "field": "destination.address"
      }
    ]
  },
  {
    "value": "Connection from {source.ip} to {destination.address}",
    "conditions": [
      {
        "field": "event.category",
	"value": "network"
      },
      {
        "field": "event.type",
	"value": "connection"
      },
      {
        "field": "source.ip"
      },
      {
        "field": "destination.address"
      }
    ]
  },
```